tests/test-values.yaml

```
@@ -0,0 +1,220 @@
+  enabled: true
+  networkPolicy:
+    enabled: true
+    flavor: kubernetes
+    discovery:
+      port: 3100
+  gateway:
+    extraContainers:
+    - args:
+      - --listen
+      - 127.0.0.1:8053
+      - --hostsfile=/etc/hosts
+      - --enable-search
+      - --verbose
+      image: gsoci.azurecr.io/giantswarm/go-dnsmasq:release-1.0.7
+      imagePullPolicy: IfNotPresent
+      name: dnsmasq
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+          - ALL
+        readOnlyRootFilesystem: true
+        seccompProfile:
+          type: RuntimeDefault
+    ingress:
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt-giantswarm
+      enabled: true
+      hosts:
+      - host: loki.golem.gaws.gigantic.io
+        paths:
+        - path: /loki/api/v1/push
+          pathType: ImplementationSpecific
+      ingressClassName: nginx
+      tls:
+      - hosts:
+        - loki.golem.gaws.gigantic.io
+        secretName: loki-ingress-cert
+    nginxConfig:
+      customBackendUrl: http://loki-multi-tenant-proxy.loki.svc.cluster.local:3102
+      customReadUrl: http://loki-multi-tenant-proxy.loki.svc.cluster.local:3100
+      customWriteUrl: http://loki-multi-tenant-proxy.loki.svc.cluster.local:3101
+    service:
+      labels:
+        prometheus.io/service-monitor: "false"
+    verboseLogging: false
+  loki:
+    analytics:
+      reporting_enabled: false
+    compactor:
+      delete_request_store: s3
+      retention_enabled: true
+    image:
+      tag: 3.1.0
+    ingester:
+      autoforget_unhealthy: true
+      chunk_encoding: snappy
+    limits_config:
+      discover_service_name:
+      - service
+      - app
+      - application
+      - name
+      - app_kubernetes_io_name
+      - container
+      - container_name
+      - component
+      - workload
+      - job
+      - scrape_job
+      ingestion_burst_size_mb: 100
+      ingestion_rate_mb: 50
+      max_global_streams_per_user: 10000
+      max_line_size: 10MB
+      retention_period: 744h
+      shard_streams:
+        enabled: true
+      split_queries_by_interval: 12h
+      volume_enabled: true
+    pattern_ingester:
+      enabled: true
+    rulerConfig:
+      remote_write:
+        add_org_id_header: false
+        clients:
+          mimir:
+            url: http://mimir-gateway.mimir/api/v1/push
+        enabled: true
+      wal:
+        dir: /var/loki/ruler-wal
+    schemaConfig:
+      configs:
+      - from: "2023-01-24"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v12
+        store: boltdb-shipper
+      - from: "2024-01-02"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v12
+        store: tsdb
+      - from: "2024-05-23"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v13
+        store: tsdb
```

We should ditch the old schemaconfigs.

```suggestion
      - from: "2024-05-23"
        index:
          period: 24h
          prefix: loki_index_
        object_store: s3
        schema: v13
        store: tsdb
```

---

Fine with me, I removed those.